### PR TITLE
Added Basic Array Questions

### PR DESCRIPTION
**Pull request by ratneshrt**
This solves all the basic questions of Array using Recursion #14

### CODE REVIEW

File: recursion/Basic Array Questions/Max_arr.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Program to find maximum element in an array using recursion
+
+def max_arr_rec(a,n):
+    if n==1: # Base Case if array containing only one element then return it
+        return a[0]
+    else:
+        previous = max_arr_rec(a,n-1)
```
Comment (fms12) [inline recursion/Basic Array Questions/Max_arr.py:7]: please add comments for better understanding. follow as same in other code snippet

**Review by fms12 — CHANGES_REQUESTED** [review]

**Review by ratneshrt — COMMENTED** [review]
Comments added to every code.

**Review by ratneshrt — COMMENTED** [review]
[UPDATED]

Added comments for better understanding of code and algorithm

**Review by ratneshrt — COMMENTED** [review]
Comments Added to First_Index.py

**Review by fms12 — APPROVED** [review]

**Pull request merged**